camunda-formio-tasklist-vue/src/styles/camundaFormIOFormList.scss

```diff
@@ -76,7 +76,7 @@
 
 .cft-height {
   overflow: scroll !important;
-  height: 93vh;
+  height: calc(100vh -200px);
```

Comment:
will using px cause any issue in small desktop screen resolution?

Reply:
its calculated value, its should be uniform